.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: scala
 
 scala:
-   - 2.11.7
+   - 2.11.12
```

Comment:
If this library update are not necessary for the support of 2019.* I would rather move to another merge request. The same for the update of the gradle plugin and scalatest.

Reply:
Done.